src/generate/interrupt.rs

```diff
@@ -125,7 +125,35 @@ pub fn render(
                     ];
             });
         }
-        Target::RISCV => {}
+        Target::RISCV => {
+            for name in &names {
+                writeln!(
+                    device_x,
+                    "PROVIDE({} = DefaultExternalInterruptHandler);",
```

Comment:
Do we need a separate handler for this? I think we can reuse `DefaultInterruptHandler`.

Reply:
I just tested it and that works too. I'm not sure if it makes sense to have them point to the same handler.

Maybe the interrupt controller handler needs to do some housekeeping even if there is no handler declared for this interrupt line. 

The intention with this different symbol was to allow the hal to implement its own interrupt controller, since as of now, the IRQ management of riscv is not well fixed, where it is for NVIC/ARM.

For my soc pointing to the same handler is fine, but i don't know if that might be to restrictive for some implementation...

Reply:
I have no idea what kind of housekeeping can be done in this case. If there is a need for such, maybe it can be moved to the interrupt controller driver out of the handler functions.

Reply:
I'm fine switching it, we'll see if anyone needs that kind of functionality later